pkg/messageformat/parse_test.go

```diff
@@ -1,136 +1,233 @@
 package messageformat
 
 import (
-	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+//nolint:lll
 func Test_Parse(t *testing.T) {
 	t.Parallel()
 
 	for _, test := range []struct {
-		name, input string
-		expectedErr error
-		expected    []interface{}
+		name     string
+		input    string
+		expected AST
 	}{
+		// singular tests
 		{
 			name:     "empty",
 			input:    "",
-			expected: []interface{}(nil),
+			expected: AST(nil),
 		},
 		{
 			name:     "empty expr",
 			input:    "{}",
```

Comment:
If this is considered an expression -> 
An expression MUST NOT be empty.
https://github.com/unicode-org/message-format-wg/blob/main/spec/syntax.md#expressions

Reply:
yeah it seems so

Reply:
deleted, as it wont happen IRL